t/17-build_tagging.t

```diff
@@ -64,11 +87,12 @@ subtest 'tag icon on group overview on important build' => sub {
 };
 
 subtest 'test whether tags with @ work, too' => sub {
-    post_comment_1001 'tag:0048@0815:important:GM';
+    post_comment_1001 'tag:0048@0815:important:RC2';
     my $get  = $t->get_ok('/group_overview/1001')->status_is(200);
     my @tags = $t->tx->res->dom->find('.tag')->map('text')->each;
-    is(scalar @tags, 2,    'one build tagged');
-    is($tags[0],     'GM', 'tag description shown');
+    is(scalar @tags, 2,    'two builds tagged');
+    # this build will sort *above* the first build, so item 0
+    is($tags[0],     'RC2', 'tag description shown');
```

Comment:
To explain this change: the existing test uses the same description ('GM') for both test tags, which seemed to me to be potentially prone to false passes - if somehow the wrong tag description was applied to each build, or something, we wouldn't notice it. So I changed this so that every test tag (these two, plus the two Fedora ones later) uses a different description, so we can be totally sure each build is listed with its own tag.